### PR TITLE
Abdul/temp force sensor

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for TempForceSensor app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : TempSensor.csproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise
    Added the BIOS setting for UPBOARD Pin configuration
    Added the app *packages installing steps

### CODE REVIEW

File: TempForceSensor/CS/README.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+* MCP3002 or MCP3208
+
+Below are the pinouts of the MCP3002 and MCP3208 analog-to-digital converters (ADC) used in this sample.
+
+![Electrical Components](../../Resources/MCP3002.png)
+![Electrical Components](../../Resources/MCP3208.png)
+
+* UP Board
+
+ ![UP Board Pinout](../../Resources/UP Board_Pinout.png)
```
Comment (rsameser) [inline TempForceSensor/CS/README.md]: Make sure this picture is visible

**Review by rsameser — CHANGES_REQUESTED** [review]
Ensure images are able to populate

File: TempForceSensor/CS/README.md (hunk 1 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/README.md]: File path of the image is updated, It is visible now.

**Review by rsameser — APPROVED** [review]

File: TempForceSensor/CS/MainPage.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+            InitSPI();
+        }
+
+        private async void InitSPI()
+        {
+            try
+            {
+                var settings = new SpiConnectionSettings(SPI_CHIP_SELECT_LINE);
+                settings.ClockFrequency = 1000000;// For Upboard use 1MHzs and For Rpi use 500KHzs
```
Comment (neilsh-msft) [inline TempForceSensor/CS/MainPage.xaml.cs]: Change comment from "// For Upboard use 1MHzs and For Rpi use 500KHzs"
to "// For UP Board use 1MHz and For Rpi use 500KHz"

File: TempForceSensor/CS/MainPage.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+                         result <<= 8;
+                         result += data[2];
+                    }
+                    break;
+            }
+
+            return result;
+        }
+
+        /*RaspBerry Pi2  Parameters*/
```
Comment (neilsh-msft) [inline TempForceSensor/CS/MainPage.xaml.cs]: RaspBerry Pi2  - change to "UP Board"

File: TempForceSensor/CS/MainPage.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+        private const string SPI_CONTROLLER_NAME = "SPI2";  /* For Upboard, use SPI2 and For Raspberry Pi use SPI0      */
+        private const Int32 SPI_CHIP_SELECT_LINE = 0;       /* Line 0 maps to physical pin number 24 on the Upboard     */
```
Comment (neilsh-msft) [inline TempForceSensor/CS/MainPage.xaml.cs]: "UP Board" not "Upboard".

File: TempForceSensor/CS/README.md [hunk 5]
```diff
@@ -0,0 +1,234 @@
...
+
+You can also use a Force sensor in this sample. Try to press the Force sensor gentle or hard to see the data output difference.
+
+This sample only has C# version.
+
+## Read before start
+This sample assumes that UP Board has already been set up as below:
+
+- UP Board has been connected to HDMI monitor
+- A SD image card has been plugged to UP Board
```
Comment (neilsh-msft) [inline TempForceSensor/CS/README.md]: Does UP Board even have an SD card slot?  This line should be removed I think.

File: TempForceSensor/CS/README.md [hunk 6]
```diff
@@ -0,0 +1,234 @@
...
+```
+
+* Initialize SPI pin
+```csharp
+private async void InitSPI()
+{
+    try
+    {
+        var settings = new SpiConnectionSettings(SPI_CHIP_SELECT_LINE);
+        settings.ClockFrequency = 1000000;// For UP Board use 1MHzs and For Rpi use 500KHzs
```
Comment (neilsh-msft) [inline TempForceSensor/CS/README.md]: "MHzs" should just be "MHz".

File: TempForceSensor/README.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+```
+
+* Initialize SPI pin
+```csharp
+private async void InitSPI()
+{
+    try
+    {
+        var settings = new SpiConnectionSettings(SPI_CHIP_SELECT_LINE);
+        settings.ClockFrequency = 1000000;// For UP Board use 1MHzs and For Rpi use 500KHzs
```
Comment (neilsh-msft) [inline TempForceSensor/README.md]: "MHzs" should be "MHz"

File: TempForceSensor/README.md [hunk 8]
```diff
@@ -0,0 +1,234 @@
...
+        settings.Mode = SpiMode.Mode0; //Mode3;
+
+        string spiAqs = SpiDevice.GetDeviceSelector(SPI_CONTROLLER_NAME);
```
Comment (neilsh-msft) [inline TempForceSensor/README.md:130]: The actual code doesn't use this pattern.  Instead it uses SpiDevice.GetDefaultAsync() and assumes that the device returned is the correct one even though the source code has an "SPI_CONTOLLER_NAME" constant define.  Should this pattern be applied to the mainpage.xaml.cs file?

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

File: TempForceSensor/CS/MainPage.xaml.cs (hunk 2 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/MainPage.xaml.cs]: Updated the Changes

File: TempForceSensor/CS/MainPage.xaml.cs (hunk 3 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/MainPage.xaml.cs]: Updated the Changes

File: TempForceSensor/CS/MainPage.xaml.cs (hunk 4 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/MainPage.xaml.cs]: Updated the Changes

File: TempForceSensor/CS/README.md (hunk 5 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/README.md]: Removed the line

File: TempForceSensor/CS/README.md (hunk 6 above)
Comment (v-abdrahman) [reply TempForceSensor/CS/README.md]: Updated the Changes

File: TempForceSensor/README.md (hunk 7 above)
Comment (v-abdrahman) [reply TempForceSensor/README.md]: Updated the Changes

File: TempForceSensor/README.md (hunk 8 above)
Comment (v-abdrahman) [reply TempForceSensor/README.md:130]: Available SPI devices gets recognize automatically in UPBOARD
SpiDevice.GetDeviceSelector();

As UPBOARD has SPI2 mode available, so we are updating that in the code.

**Pull request merged**